src/AssetPack/AssetPack.router.ts

```diff
@@ -100,22 +100,43 @@ export class AssetPackRouter extends Router {
 
   getAssetPacks = async (req: AuthRequest) => {
     const ethAddress = req.auth ? req.auth.ethAddress : ''
-    let assetPacks: FullAssetPackAttributes[] = []
+    const owner = req.query.owner
```

Comment:
I would lowercase `req.query.owner` to sopport checksummed addresses too

Reply:
WE can do it in a reutilizable middleware

Reply:
I'll lowercase it!
How would the middleware be? Will it just lowercase an address? Do we need it somewhere else?